src/components/Popup/Popup.tsx

```diff
@@ -266,16 +269,23 @@ export default class Popup extends AutoControlledComponent<Extendable<PopupProps
 
     const focusTrapProps = {
       ...(typeof accessibility.focusTrap === 'boolean' ? {} : accessibility.focusTrap),
-      ...popupContentAttributes,
+      ...popupWrapperAttributes,
     } as FocusTrapZoneProps
 
-    const popupContent = Popup.Content.create(content, {
-      /**
-       * if there is no focus trap wrapper, we should apply
-       * HTML attributes and positioning to popup content directly
-       */
-      defaultProps: accessibility.focusTrap ? {} : popupContentAttributes,
-    })
+    /**
+     * if there is no focus trap wrapper, we should apply
+     * HTML attributes and positioning to popup content directly
+     */
+    const popupContentAttributes = accessibility.focusTrap ? {} : popupWrapperAttributes
+
+    const popupContent = React.isValidElement(content)
+      ? React.cloneElement(content, {
+          ...content.props,
```

Comment:
Why we need a spread there? React will add missing properties: https://codesandbox.io/s/9ornj4qrko

```suggestion
React.cloneElement(content, popupContentAttributes)
```